database/src/migrations/release.0.10/smoketest_release.sql

```diff
@@ -0,0 +1,144 @@
+-- smoketest_release.sql
+-- run as db super user
+\c biohub
+
+do $$
+declare
+  __count integer = 0;
+begin  
+  set role postgres;
+  set search_path=biohub;
+
+  delete from system_user where user_identifier = 'myIDIR';
+
+  insert into system_user (uis_id, user_identifier, record_effective_date, create_date, create_user) select id, 'myIDIR', now(), now(), 1 from user_identity_source where name = 'IDIR' and record_end_date is null;
+
+  select count(1) into __count from system_user;
+<<<<<<< HEAD
+  assert __count > 1, 'FAIL system_user';
+=======
+  assert __count = 2, 'FAIL system_user';
+>>>>>>> a39d7d62c5023a2196dac997b6878e6f1dd4e5b0
```

Comment:
Looks like some leftover merge artifacts `>>>>` etc

Reply:
doesn't matter, just a piece of of smoke test code